types/lib/query-interface.d.ts

```diff
@@ -553,7 +557,8 @@ export class QueryInterface {
     returnType: string,
     language: string,
     body: string,
-    options?: QueryOptions
+    optionsArray?: string[],
+    options?: QueryOptionsWithForce
```

Comment:
Looks like you also found (and fixed!) a bug in typings, right? Where it was previously `QueryOptions` you fixed it to `string[]`. Nice work.

Reply:
Yeah :+1: